include/qdmi/constants.h

```diff
@@ -48,7 +48,7 @@ enum QDMI_STATUS {
   QDMI_ERROR_NOTSUPPORTED = -9,     ///< Operation is not supported.
   /// Resource is in the wrong state for the operation.
   QDMI_ERROR_BADSTATE = -10,
-  QDMI_ERROR_TIMEOUT = -11, ///< Operation timed out.
+  QDMI_ERROR_TIMEOUT = -11 ///< Operation timed out.
```

Comment:
```suggestion
  QDMI_ERROR_TIMEOUT = -11, ///< Operation timed out.
```
Seems like an unintentional change. I'd rather keep the comma as this way adding a new enum member does not cause changes in unrelated lines.

Reply:
I would happily agree if that would not be a C99 feature (and is prohibited in strict C98 and hence causes a compiler warning) and all other enums do not do that. What do you say?

Reply:
We use a way more modern version of C than all of that. So it is definitely allowed. If this is throwing a warning, we are not setting a flag correctly somewhere that sets the C standard.

Reply:
Ok, then I saw the issue in some other project, where I included QDMI. Then it is probably the problem of the setup in the other project.